### DISCUSSION TITLE
Are Pull Requests and Actions down/degraded?

### DISCUSSION THREAD

**Discussion by irwinb** [community/community#75182] [Actions]
### Select Topic Area

Bug

### Body

The following has not been working for me starting about 30 minutes ago:
* GitHub auto-detecting that I pushed to a new branch and show a button to create a new pull request
* Pushing a commit to a branch with an open pull request does not trigger the pull request to refresh
* Pull request is stuck in "Checking for ability to merge automatically…"
* Manually trying to trigger an Action doesn't work. I just get a banner saying the run was successfully requested.

I checked all my orgs usage limits etc, everything looks good. I haven't changed any configuration.

**Comment by MitchTalmadge** [comment]
I think so. My latest release is not showing up, and my workflow did not trigger from the tag I created.

Also pages are degraded: https://www.githubstatus.com/incidents/wyk0ns67krlz

**Comment by andrzejressel** [comment]
In my case also list of branches is outdated and force pushing does not change anything in pr - commit ID stays the same.

**Comment by digikata** [comment]
For me a push of a new commit onto a PR branch succeeds at the command line, but the repo PR page does not show it, nor does gh actions pick up the commit.

**Comment by irwinb** [comment]
FYI status page has updated https://www.githubstatus.com/

**Comment by irwinb** [comment]
It's resolved.

**Comment by pkit** [comment]
Same problem again.
"Checking for ability to merge automatically…" is stuck for 2 minutes.
Then it's not starting runners...

**Comment by sammaniamsam** [comment]
I'm also experiencing this issue as of today.
![Screenshot 2023-11-27 at 1 56 37 PM](https://github.com/community/community/assets/16583147/8fb13a20-4a71-4ca7-8178-409c7859329d)

**Comment by suchakra-texada** [comment]
Still not resolved.

**Reply by CatcherInThePy** [reply]
Same here. Experiencing the same issue as of April 24, 2024. Re-logging in or clearing browser cache did not work, which are some of the previously suggested solutions online. Nothing on the status page though. (https://www.githubstatus.com/)

**Reply by rahul-2819** [reply]
facing same issue from morning

**Reply by zzhai-ng** [reply]
Same issue since this morning

**Reply by bbmena** [reply]
Same issue as well. Stuck for over an hour now

**Comment by gio-vlasov** [comment]
Happened to me several times today

**Comment by TusharShahi** [comment]
Facing this

**Comment by akashsolanki** [comment]
Facing this exact issue.. Can someone please help?

**Comment by pkit** [comment]
Yup, btw "degraded" == "completely down" :)
